src/backend/rest.ts

```diff
@@ -362,4 +371,42 @@ export default class RestApiBackend implements Backend {
       return new MutableDataFrame();
     }
   }
+
+  async getAutocompleteBackend(): Promise<BACKEND_TYPE> {
+    return this.api<RestApiAutocompleteResponse>({
+      url: `/objects/autocomplete/sites`,
+      method: 'POST',
+      data: { value: '', parameters: {} },
+    })
+      .then(() => {
+        return BACKEND_TYPE.REST;
+      })
+      .catch(() => {
+        return BACKEND_TYPE.WEB;
+      });
+  }
+
+  async autocompleterRequest(
+    api_url = '',
+    data: unknown
```

Comment:
coulnd't you use `{ident: string, value:unkown, params:unkown}` here, and get past those any warnings?

Reply:
Yes, it was better than using any.